src/main/java/com/flansmod/common/guns/InventoryHelper.java

```diff
@@ -41,7 +41,21 @@ public static boolean addItemStackToInventory(IInventory inventory, ItemStack st
 	            			is.setItemDamage(itemDamageDiff);
 	            			return true;
 	            		} else {
-	            			is.setItemDamage(0);
+	            			is.setItemDamage(0);      
```

Comment:
Where is addItemStackToInventory called? Not an issue, I'd just like to check what this is likely to affect

Reply:
In 'ItemGun' on reload() and by vehicles, but they are not affected by this change since 'combineAmmo' is always false for those.